subt/docker/robotika/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM robotika/subt-base:2020-09-22
 
-ADD http://osgar.robotika.cz/subt/virtual/model/mdnet3.64.64.13.4.relu.pth .
-ADD http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/frozen_inference_graph.pb .
-ADD http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/cv_graph.pbtxt .
+RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/mdnet3.64.64.13.4.relu.pth
```

Comment:
@jisa suggested to add comment here about docker handling hashes ... thanks

Reply:
Done in 553f4e4